web/pgadmin/tools/erd/static/js/erd_tool/components/ERDTool.jsx

```diff
@@ -510,6 +510,10 @@ export default class ERDTool extends React.Component {
         this.diagram.getSelectedLinks().forEach((link)=>{
           this.diagram.removeOneToManyLink(link);
         });
+        if (this.diagram.getNodesData().length === 0){
```

Comment:
when we open a new ERD tool, the save button is initially disabled. It becomes enabled when we create objects or tables, but if we delete all the objects we've created, the save button should be disabled again, just like it is in the query tool. Since it wasn’t getting disabled automatically, that's the reason I added this check.